conbench/templates/index.html

```diff
@@ -3,30 +3,28 @@
 {% block app_content %}
 <div class="row">
   <div class="col-md-12">
-
-<div class="jumbotron">
-<div class="media">
-  <div class="media-body">
-    <h4 class="media-heading">Con·bench</h4>
-     <h4>/kənˈben(t)SH/</h4>
-     <h5><i>noun</i></h5>
-     <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
-  </div>
-  <div class="media-right">
-     <img class="media-object" style="padding: 5px" src="{{ url_for('static', filename='bar-graph.png') }}" height="150">
-  </div>
-</div>
-</div>
-
+    <div class="jumbotron">
+      <div class="media">
+        <div class="media-body">
+          <h4 class="media-heading">Con·bench</h4>
+          <h4>/kənˈben(t)SH/</h4>
+          <h5><i>noun</i></h5>
+          <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
+        </div>
+        <div class="media-right">
+          <img class="media-object" style="padding: 5px" src="{{ url_for('static', filename='bar-graph.png') }}" height="150">
+        </div>
+      </div>
+    </div>
   </div>
 </div>
 
 
 <div class="row">
+    <hr />
     <div class="col-md-12">
+        <h4>Recent runs</h4>
         <table id="runs" class="table table-striped table-bordered table-hover">
-          <caption>Runs
-          </caption>
```

Comment:
captions started to appear _below_ the table. This may be useful in many contexts; but here in our case we need a heading above the table. The caption was semantically misused. Can explain details, as in fine-print (like long-form figure caption).